app/forms/two_factor_options_form.rb

```diff
@@ -47,15 +43,26 @@ def update_otp_delivery_preference_for_user
     UpdateUser.new(user: user, attributes: user_attributes).call
   end
 
-  def multiple_mfa_options_enabled?
-    IdentityConfig.store.select_multiple_mfa_options
-  end
-
   def phone_selected?
     selection.include?('phone') || selection.include?('voice') || selection.include?('sms')
   end
 
   def phone_only_mfa_method?
     MfaContext.new(user).enabled_mfa_methods_count == 0
   end
+
+  def phone_alternative_enabled?
+    count = MfaContext.new(user).enabled_mfa_methods_count
+    if count >= 2 || count == 1 && MfaContext.new(user).phone_configurations.none?
+      return true
+    else
+      return false
+    end
```

Comment:
For mixed operators like `||` and `&&`, can we add parentheses around the parts which are meant to be tested together? I don't know the order of operations off the top of my head, so this is hard for me to interpret.

Also, we could return the boolean result directly, rather than extra return statement.

```suggestion
    count >= 2 || (count == 1 && MfaContext.new(user).phone_configurations.none?)
```